src/sentry/conf/server.py

```diff
@@ -1213,3 +1213,5 @@ def get_raven_config():
     # The Ruby SDK used to go by the name 'sentry-raven'...
     'sentry-raven': 'raven-ruby',
 }
+
+SOUTH_TESTS_MIGRATE = os.environ.get('SOUTH_TESTS_MIGRATE', '0') == '1'
```

Comment:
Unclear if this will work because of:

https://github.com/getsentry/sentry/blob/master/src/sentry/utils/pytest/sentry.py#L18-L44